index.html

```diff
@@ -2,11 +2,159 @@
+         <ul>
+             <li>
+                  <a href="blank.html"><img src="" width="267" height="267" alt="Сливочное с апельсиновым джемом">
+                  <p>310 ₽<span>/кг</span></p>
+                  <p>Сливочное с апельсиновым джемом и цитрусовой стружкой</p></a>
+             </li>
+             <li>
+                  <a href="blank.html"><img src="" width="267" height="267" alt="Сливочно-кофейное с кусочками шоколада">
+                  <p>380 ₽<span>/кг</span></p>
+                  <p>Сливочно-кофейное с кусочками шоколада</p></a>
+            </li>
+            <li>
+                 <a href="blank.html"><img src="" width="267" height="267" alt="Сливочно-клубничное с присыпкой из белого шоколада">
+                 <p>355 ₽<span>/кг</span></p>
+                 <p>Сливочно-клубничное с присыпкой из белого шоколада</p></a>
+            </li>
+            <li>
+                 <a href="blank.html"><img src="" width="267" height="267" alt="Сливочное крем-брюле с карамельной подливкой">
+                 <p>415 ₽<span>/кг</span></p>
+                 <p>Сливочное крем-брюле с карамельной подливкой </p></a>
+            </li>
+          </ul>
+      </section>
+     <section>
+        <!--качество продукции-->
+        <h3>Магазин Глэйси – это онлайн и офлайн-магазин по продаже мороженого собственного производства на развес</h3>
+          <ul>
+              <li>
+                   Все наше мороженое изготавливается на собственном производстве, с использованием современного оборудования и проверенных временем технологий.
+              </li>
+              <li>
+                   Для приготовления мороженого используются настоящие сливки и молоко высочайшего качества. Все дополнительные ингредиенты и добавки произведены из натурального,<br>
+                   экологически чистого сырья.
+             </li>
+             <li>
+                  Закупка ингредиентов производится только у проверенных фермерских хозяйств и компаний, с которыми нас связывает долговременное сотрудничество.
+             </li>
+             <li>
+                  Доставка нашего мороженого до заказчиков осуществляется в специальном термопаке, который не дает мороженому растаять в пути и позволяет сохранить превосходный вкус.
+             </li>
+           </ul>
+       </section>
+     <section>
+         <!--Новости -->
+         <p>Новое в нашем блоге</p>
+         <a href="blank.html">10 способов сервировки фруктовых щербетов к столу</a>
+       </section>
+     <section>
+         <!--отправка подписки -->
+         <form class="" action="https://echo.htmlacademy.ru" method="post">
+           <label for="mail"></label>
+           <p>Подпишитесь на нашу сладкую рассылку и будьте всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)</p>
+           <input type="email" id="mail" placeholder="email@example.com">
+           <button type="submit" value="ОТПРАВИТЬ">
```

Comment:
Тег `button` парный. У него есть закрывающий тег.
```suggestion
           <button type="submit">Отправить</button>
```